guibot/region.py

```diff
@@ -740,18 +825,22 @@ def click_vanish(self, click_image_or_location: Image | Location,
                     raise error
         return self
 
-    def click_at_index(self, anchor: str | Target, index: int = 0,
-                       find_number: int = 3, timeout: int = 10) -> "Match":
+    def click_at_index(
+        self,
+        anchor: str | Target,
+        index: int = 0,
+        find_number: int = 3,
+        timeout: int = 10,
+    ) -> "Match":
         """
-        Find all instances of an anchor image and click on the one with the
-        desired index given that they are horizontally then vertically sorted.
+        Find and click on a specific instance of an anchor image, given their horizontal and vertical sorting.
```

Comment:
Find and click on a specific instance of an anchor image indexed by horizontal and vertical sorting.

Reply:
Done